docs/User-Guide_Board-Support-Rules.md

```diff
@@ -82,6 +82,8 @@ For an SBC to be considered supported:
 
 Additional Caveats:
 
+* R&D bills paid. Failsafe in case Armbian members or a team was hired
+* support bills covered by vendor or end users with crowdfunding campaign
```

Comment:
```suggestion
* If the burden placed on the Armbian team is too high funding maybe needed to assure:
    -  R&D bills paid. Failsafe in case Armbian members or a team was hired
    -  support bills covered by vendor or end users with crowdfunding campaign
```

Reply:
This might make things more clear?

Reply:
> If the burden placed on the Armbian team is too high

Burden is not too high only if everything is paid and everyone is enjoying. Which was never. Even in your paid job, you can easily get to the point that burden is too high. Or to stop enjoying.

Reply:
So really what we IMHO want is that if they're too much work for standard support that they need to goto gold rather than try to add line items into standard.

Anyway adjust how you see fit.  Let's get merged and move forward